backend/app/helpers/spree/admin/stores_helper.rb

```diff
@@ -0,0 +1,18 @@
+module Spree
+  module Admin
+    module StoresHelper
+      def selected_checkout_zone(store)
+        return store.checkout_zone_id unless store.checkout_zone_id.nil?
```

Comment:
This method can evaluate to 3 different data types:

`Integer` whet this line matches
`Spree::Store` when lines below match
`nil` when no condition is satisfied or no zone is found by name

@przemosk can we do something about it, please?

Reply:
@nnande as we discussed internally returned type is one 🎉  👼🏻

Reply:
This is not the issue @przemosk